packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -235,10 +237,28 @@ Provide a consis summary in 100 words or less.
 			state.postForecastRunId = postSimId;
 			emit('update-state', state);
 
+			const datasetName = `Forecast run ${state.postForecastRunId}`;
+			const projectId = useProjects().activeProject.value?.id ?? '';
```

Comment:
We have an [activeProjectId](https://github.com/DARPA-ASKEM/terarium/blob/main/packages/client/hmi-client/src/composables/activeProject.ts#L5)
```suggestion
			const projectId = useProjects().activeProjectId;
```